app/src/main/java/com/codesoom/assignment/handler/TodoHttpHandler.java

```diff
@@ -0,0 +1,62 @@
+package com.codesoom.assignment.handler;
+
+import com.codesoom.assignment.Exception.NotFoundException;
+import com.codesoom.assignment.service.TodoService;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import java.io.*;
+import java.net.URI;
+import java.util.stream.Collectors;
+
+public class TodoHttpHandler implements HttpHandler {
+    private TodoService todoService = new TodoService();
+    @Override
+    public void handle(HttpExchange exchange) throws IOException {
+        String method = exchange.getRequestMethod();
+        String content = "Hello World";
+        Long taskId = 0L;
+        URI uri = exchange.getRequestURI();
+        String path = uri.getPath();
+
+        InputStream inputStream = exchange.getRequestBody();
+        String body = new BufferedReader(new InputStreamReader(inputStream))
+                .lines()
+                .collect(Collectors.joining("\n"));
+        System.out.println(method + " " + path);
+        try{
+            if (method.equals("GET") && path.equals("/tasks")) {
```

Comment:
이렇게 하면 `method`가 null 일 경우에 NullPointerException 위험이 있습니다. 그래서 보통은 이렇게 써주죠.

```suggestion
            if ("GET".equals(method) && path.equals("/tasks")) {
```

Reply:
NullPointerException... 조심하겠습니다!!